src/main/java/knightminer/simplytea/data/gen/ShapelessHoneyRecipe.java

```diff
@@ -31,7 +33,7 @@ public class ShapelessHoneyRecipe extends ShapelessRecipe {
 	private final Ingredient honey;
 	private final String tag;
 	public ShapelessHoneyRecipe(ResourceLocation id, String group, ItemLike tea, Ingredient honey, String tag) {
-		super(id, group, TeaCupItem.withHoney(new ItemStack(tea), tag), NonNullList.of(Ingredient.EMPTY, Ingredient.of(tea), honey));
+		super(id, group, CraftingBookCategory.MISC, TeaCupItem.withHoney(new ItemStack(tea), tag), NonNullList.of(Ingredient.EMPTY, Ingredient.of(tea), honey));
```

Comment:
Food also makes more sense for this category; arguably should be just passed in from JSON however shapeless recipes do it if thats not too much effort.

Reply:
CraftingBookCategory only has 4 values: BUILDING, REDSTONE, EQUIPMENT and MISC